cmd/btfs/daemon.go

```diff
@@ -311,6 +313,21 @@ func daemonFunc(req *cmds.Request, re cmds.ResponseEmitter, env cmds.Environment
 		return err
 	}
 
+	// init ledger account and get balance
+	go func() {
+		bo := backoff.WithMaxRetries(backoff.NewExponentialBackOff(), 3)
+		err := backoff.Retry(func() error {
+			balance, err := escrow.Balance(req.Context, cfg)
+			if err == nil {
+				fmt.Println("banlance:", balance)
+			}
+			return err
+		}, bo)
+		if err != nil {
+			log.Error("Get balance failed.")
```

Comment:
Change to "Unable to get Ledger balance. Please check network settings and restart daemon."

Reply:
Log the error as well.